src/layouts/default.vue

```diff
@@ -5,25 +5,42 @@
     <main class="body">
       <section class="title">
         <section class="title-top">
-          <button
-            v-if="isChild"
-            class="btn role-link btn-sm btn-back"
-            type="button"
-            @click="routeBack"
+          <transition-group
+            name="fade-group"
+            class="title-group"
+            appear
           >
-            <span
-              class="icon icon-chevron-left"
-            />
-          </button>
-          <h1 data-test="mainTitle">
-            {{ title }}
-          </h1>
-          <section
-            v-if="action"
-            class="actions"
+            <button
+              v-if="isChild"
+              key="back-btn"
+              class="btn role-link btn-sm btn-back fade-group-item"
+              type="button"
```

Comment:
Can we add a test element `data-test="back-btn"`?
It can be handy to test on E2E specs :D

Reply:
Sounds easy enough, thanks for calling this out!